src/lib/exporter/ffmpegExporter.ts

```
@@ -0,0 +1,408 @@
+						const timestamp = frameIndex * frameDurationUs; // microseconds here
+						webcamFrame = webcamFrameQueue ? await webcamFrameQueue.dequeue() : null;
+						if (this.cancelled) {
+							return;
+						}
+
+						const sourceTimestampUs = sourceTimestampMs * 1000;
+						await renderer.renderFrame(videoFrame, sourceTimestampUs, webcamFrame);
+						const canvas = renderer.getCanvas();
+
+						// Fastest path in existence: GPU texture -> Hardware H264 Encoder
+						const timestamp = frameIndex * frameDurationUs;
```

Biome (2.4.10)

[error] Shouldn't redeclare 'timestamp'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)